tests/test_model.py

```diff
@@ -1202,7 +1325,35 @@ def test_bad_explicit_model_implicit_data(train_val_implicit_sample_data):
     model = MatrixFactorizationModel(train=train,
                                      val=val,
                                      loss='mse')
-    trainer = CollieTrainer(model=model, logger=False, checkpoint_callback=False, max_epochs=1)
+    trainer = CollieTrainer(model=model, logger=False, enable_checkpointing=False, max_epochs=1)
 
     with pytest.raises(ValueError):
         trainer.fit(model)
+
+
+def test_get_item_preds_err(implicit_model,
+                            train_val_implicit_data):
+    model = implicit_model
+    train, val = train_val_implicit_data
+
+    with pytest.raises(
+        ValueError,
+        match=f'``user_id`` 11111 is not in the model. Expected id between 0 and {train.num_users}'
+    ):
+        model.get_item_predictions(user_id=11111,
```

Comment:
Just in case the underlying dataset of the tests ever change, can we dynamically set this to be something like: 
```suggestion
        model.get_item_predictions(user_id=train.num_users + 1,
```

Reply:
Duh! Way better :)